app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) == 3 and parts[0] == "cp":
+        src_file, dest_file = parts[1], parts[2]
+
+        if src_file != dest_file:
+            try:
+                with (
+                    open(src_file, "r") as file_in,
+                    open(dest_file, "w") as file_out
+                ):
+                    file_out.write(file_in.read())
+            except (FileNotFoundError, IOError) as e:
```

Comment:
FilenotFoundError is a sublcass for OSError, IOError is an alias for OSError - so you can simply use OSError here and it will cover all errors that might happen during file handling